internal/pkg/cli/deploy/lbws.go

```diff
@@ -246,7 +246,7 @@ func (d *lbWebSvcDeployer) validateALBRuntime() error {
 }
 
 func (d *lbWebSvcDeployer) validateNLBRuntime() error {
-	if d.lbMft.NLBConfig.Aliases.IsEmpty() {
+	if d.lbMft.NLBConfig.PrimaryRoutingRule.Aliases.IsEmpty() {
```

Comment:
shall we check additional routing rules too? Or they can't exist without primary routing rule

Reply:
Additional rules won't exist before primary routing rule.

Reply:
Yes, we error out because we check for `nlb.port` required parameter is defined or not? If it not defined then we error out saying defined required `nlb.port` parameter.

Reply:
gotcha sounds good then